src/acl/acl.ts

```diff
@@ -142,20 +142,17 @@ export function internal_getContainerPath(resourcePath: string): string {
 }
 
 /**
- * Verify whether an ACL was found for the given Resource.
+ * Verify whether the given Resource has a resource ACL attached.
```

Comment:
Just as an FYI --  some people go by the rule that the description should be a declarative sentence "Verifies ..." rather than imperative "Verify ..."

I've never been particularly strict about it one way or the other as long as people are consistent - as I've never found people to be confused by the use of one over the other.  My rule is if everything else is clear, no one cares as long as we're consistent.

Reply:
No strong preference here, and agreed with the consistency - feel free to change it to whatever you feel is best (and let us know which form to use in the future).

Reply:
I'll update to using declarative as I go through the API docs since sooner or later, we'll get feedback from a user stating we should be using the declarative - and we should going forward use the declarative  -- just to be consistent not just within this library but across libraries.